core/src/main/scala/transport/netty/Transport.scala

```diff
@@ -177,9 +177,9 @@ object Enframe extends ChannelOutboundHandlerAdapter {
     obj match {
       case Bits(bv) =>
         val byv = bv.toByteVector
```

Comment:
@timperrett This is only here to see if PR builds successfully on Travis, I'll rebase on master when PR that @sbuzzard open is merged.